program/src/main/System.vala

```diff
@@ -101,35 +99,46 @@ public class System
             var menuItemQuestion = new Gtk.MenuItem.with_label("Question");
             menuItemQuestion.set_submenu(menuQuestion);
             menuBar.append(menuItemQuestion);
+
         }
         
-        
+
         //setup window area UI
         {
             //setup the exam view on left, and tools on right
             var hpaned = new Gtk.Paned(Gtk.Orientation.HORIZONTAL);
             hpaned.set_size_request(800, 400); //minimum size of entire window content
             menuPane.pack2(hpaned, true, false);
             
-            //left side: exam view
+            //left side: exam view AND progressBar?
+            var viewPane = new Gtk.Paned(Gtk.Orientation.VERTICAL);
```

Comment:
This file hasn't been updated for coding conventions so i don't expect you to do the work to update it. But variable names should be lowercase_with_underscores